arch/risc-v/src/common/riscv_mtimer.c

```diff
@@ -84,7 +84,13 @@ static const struct oneshot_operations_s g_riscv_mtimer_ops =
 static uint64_t riscv_mtimer_get_mtime(struct riscv_mtimer_lowerhalf_s *priv)
 {
 #ifdef CONFIG_ARCH_RV64
-  return getreg64(priv->mtime);
```

Comment:
should we modify 32bit version too?

Reply:
AFAIK, there is no issue on RV32 SoCs. Shall we add this when someone reports this?